spec/dummy/app/assets/stylesheets/preview.scss

```diff
@@ -17,6 +17,10 @@ body {
   color: #222;
 }
 
+.page_header {
```

Comment:
Why is this only in `preview`? Shouldn't it be part of the page_header component?